src/phoenix/server/api/input_types/CreateSpanAnnotationsInput.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+import strawberry
+from strawberry.relay import GlobalID
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class CreateSpanAnnotationsInput:
```

Comment:
```suggestion
class CreateSpanAnnotationInput:
```

Since there is only one span annotation being created.